README.md

```diff
@@ -58,11 +66,104 @@ This library is header-only, so all the source code can be found in the `include
 |------|------|-----------|
 | Things approved for the C++ standard | `<stdexec/...>` | `::stdexec` |
 | Generic additions and extensions | `<exec/...>` | `::exec` |
-| Vendor-specific extensions and customizations | <code>&lt;<i>(vendor)</i>exec/...&gt;</code> | <code>::<i>(vendor)</i>exec</code> |
+| NVIDIA-specific extensions and customizations | <code>&lt;nvexec/...&gt;</code> | <code>::nvexec</code> |
 | | |
 
+## How to get `stdexec`
+
+There are a few ways to get `stdexec`:
+1. Clone from GitHub
+   - `git clone https://github.com/NVIDIA/stdexec.git`
+2. Download the [NVIDIA HPC SDK starting with 22.11](https://developer.nvidia.com/nvidia-hpc-sdk-releases)
+3. (Recommended) Use [CMake Package Manager (CPM)](https://github.com/cpm-cmake/CPM.cmake) to automatically pull `stdexec` as part of your CMake project. [See below](#cmake-package-manager-cpm) for more information.
+
+You can also try it directly on [godbolt.org](https://godbolt.org/z/acaE93xq3) where it is available as a C++ library or via the nvc++ compiler starting with version 22.11 ([see below](#nvhpc-sdk) for more details).
+
+## Using `stdexec`
+
+### Requirements
+
+`stdexec` requires compiling with C++20 (`-std=c++20`) but otherwise does not have any dependencies and only requires a sufficiently new compiler:
+- clang 12+
+- gcc 11+
+- [nvc++ 22.11+](https://developer.nvidia.com/nvidia-hpc-sdk-releases) (required for [GPU support](#gpu-support))
+
+How you configure your environment to use `stdexec` depends on how you got `stdexec`.
+
+### NVHPC SDK
+
+Starting with the 22.11 release of the [NVHPC SDK](https://developer.nvidia.com/nvidia-hpc-sdk-releases), `stdexec` is available as an experimental, opt-in feature. Specifying the `--experimental-stdpar` flag to `nvc++` makes the `stdexec` headers available on the include path. You can then include any `stdexec` header as normal: `#include <stdexec/...>`, `#include <nvexec/...>`.  See [godbolt example](https://godbolt.org/z/qc1h3sqEv).
+
+GPU features additionally require specifying `-stdpar=gpu`. For more details, see [GPU Support](#gpu-support).
+
+### GitHub
+
+As a header-only C++ library, technically all one needs to do is add the `stdexec` `include/` directory to your include path as `-I<stdexec root>/include` in addition to specifying any necessary compile options.
+
+For simplicity, we reocmmend using the [CMake targets](#cmake) that `stdexec` provides as they encapsulate the necessary configuration.
```

Comment:
```suggestion
For simplicity, we recommend using the [CMake targets](#cmake) that `stdexec` provides as they encapsulate the necessary configuration.
```

Reply:
Blah, the VSCode spell check extension I normally use doesn't work in the browser/codespace for some reason.